pkgs/build-support/cc-wrapper/default.nix

```diff
@@ -287,7 +287,7 @@ stdenv.mkDerivation {
     + optionalString (libc != null) (''
       touch "$out/nix-support/libc-cflags"
```

Comment:
Do you mean this? It seems to be only created if `libc != null`?

Reply:
Ah yes, it is not quite static, but still a bit simpler: if you have a libc you get a libc cflags files whether or not libc actually needs extra cflags. I think this is a good balance between easy to use, and keeping a cc wrapper without a libc agnostic of libc-related things.

Sorry for misleading you with my original comment.

Reply:
So, we could just also `touch $out/nix-support/libc-crt1-cflags` in the same conditional block.

Reply:
Ok, but what difference does it make. `>> $out/nix-support/libc-crt1-cflags` should always create this file no?

Reply:
I was touching them whether or not it was needed to make this blindly obvious, but yes I suppose since this is in the same `''...''` it is already pretty obvious.